src/ui/preview/Preview.vue

```diff
@@ -21,55 +21,42 @@
  *****************************************************************************/
 <template>
 <div class="l-preview-window">
-    <div class="l-browse-bar">
-        <div class="l-browse-bar__start">
-            <div
-                class="l-browse-bar__object-name--w"
-                :class="type.cssClass"
-            >
-                <span class="l-browse-bar__object-name">
-                    {{ domainObject.name }}
-                </span>
-                <context-menu-drop-down :object-path="objectPath" />
-            </div>
-        </div>
-        <div class="l-browse-bar__end">
-            <div class="l-browse-bar__actions">
-                <view-switcher
-                    :views="views"
-                    :current-view="currentView"
-                    @setView="setView"
-                />
-            </div>
-        </div>
-    </div>
+    <PreviewHeader
+        :current-view="currentView"
+        :domain-object="domainObject"
+        :views="views"
+        @setView="setView"
+    />
     <div class="l-preview-window__object-view">
         <div ref="objectView"></div>
     </div>
 </div>
 </template>
 
 <script>
-import ContextMenuDropDown from '../../ui/components/contextMenuDropDown.vue';
-import ViewSwitcher from '../../ui/layout/ViewSwitcher.vue';
+import PreviewHeader from './preview-header.vue';
 
 export default {
     components: {
-        ContextMenuDropDown,
-        ViewSwitcher
+        PreviewHeader
     },
     inject: [
         'openmct',
         'objectPath'
     ],
+    props: {
+        notebookEnabled: {
```

Comment:
Is this used?

Reply:
@nikhilmandlik Just need to address this to get it merged into Master

Reply:
removed.